hack/verify-license.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright 2023 The Karmada Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${REPO_ROOT}"
+
+if [[ "$(which addlicense)" == "" ]]; then
+	go install github.com/google/addlicense@latest
```

Comment:
Can we pin to a stable version of `addlicense`? Such as https://github.com/google/addlicense/releases/tag/v1.1.1.

So that, any unstable changes on master won't affect our CI.